src/vs/workbench/contrib/notebook/browser/viewModel/baseCellViewModel.ts

```diff
@@ -199,7 +189,7 @@ export abstract class BaseCellViewModel extends Disposable {
 		});
 
 		this._textEditor = undefined;
-		this.textModel = undefined;
+		this.model.textModel = undefined;
```

Comment:
Should this be disposing the textModel ref? Currently that doesn't happen and it will get another ref.

Reply:
Seems wrong 
![image](https://user-images.githubusercontent.com/323878/112883456-2b823080-9083-11eb-9bcd-694e94e38808.png)